src/presenters/meta-author-presenter.php

```diff
@@ -35,12 +37,22 @@ public function present() {
 	 * @return string The author's display name.
 	 */
 	public function get() {
+		if ( $this->presentation->model->object_sub_type !== 'post' ) {
```

Comment:
I think we should use [post_type_supports](https://developer.wordpress.org/reference/functions/post_type_supports/) here:

`post_type_supports( $this->presentation->model->object_sub_type, 'author' )`